lib/src/wallet.rs

```diff
@@ -567,19 +569,19 @@ impl LightWallet {
 
     async fn add_imported_spend_key<
         WKey: WalletKey + Clone,
```

Comment:
Is `WKey` a useful abbreviation?   Should we just use `WalletKey` as the identifier?